{
  "uid" : "8038a4fd5d6af28f",
  "name" : "test_constellation_info[data1]",
  "fullName" : "TestCase.test_constellation_info.Test_Service_Info#test_constellation_info",
  "historyId" : "1c6097c37dd9566780c6e3c250c3175a",
  "time" : {
    "start" : 1630848282695,
    "stop" : 1630848282726,
    "duration" : 31
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: assert 205801 == 0",
  "statusTrace" : "self = <TestCase.test_constellation_info.Test_Service_Info object at 0x0000013EB1F3A4C8>\ndata = {'data': {'consName': '', 'type': 'today'}, 'detail': '异常测试：星座字段空', 'resp': {'error_code': 0, 'reason': 'Return Successd!', 'result': 'NAME ERROR!', 'resultcode': '201'}}\n\n    @pytest.mark.process\n    @pytest.mark.parametrize('data',testdata)\n    def test_constellation_info(self, data):\n        self.log.info('%s{%s}' % ((sys._getframe().f_code.co_name, '------开始星座运势查询接口测试-----')))\n        with allure.step('星座运势查询接口'):\n            msg = Api_News_Service().get_constellation(data['data']['consName'])\n        self.log.info('%s:%s' % ((sys._getframe().f_code.co_name, '获取请求结果：%s' % msg.json())))\n        # # 断言\n        assert data['resp']['result'] in (str)(msg.json()), '关键字\"{}\"不存在源码中!'.format(msg.json())\n        assert msg.json()['resultcode'] == data['resp']['resultcode']\n>       assert msg.json()['error_code'] == data['resp']['error_code']\nE       assert 205801 == 0\n\nTestCase\\test_constellation_info.py:29: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "base_url",
    "time" : {
      "start" : 1630848282632,
      "stop" : 1630848282632,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "stepsCount" : 0,
    "hasContent" : false
  }, {
    "name" : "_verify_url",
    "time" : {
      "start" : 1630848282632,
      "stop" : 1630848282632,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "stepsCount" : 0,
    "hasContent" : false
  }, {
    "name" : "host",
    "time" : {
      "start" : 1630848282632,
      "stop" : 1630848282632,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "stepsCount" : 0,
    "hasContent" : false
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1630848282351,
      "stop" : 1630848282632,
      "duration" : 281
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "stepsCount" : 0,
    "hasContent" : false
  } ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: assert 205801 == 0",
    "statusTrace" : "self = <TestCase.test_constellation_info.Test_Service_Info object at 0x0000013EB1F3A4C8>\ndata = {'data': {'consName': '', 'type': 'today'}, 'detail': '异常测试：星座字段空', 'resp': {'error_code': 0, 'reason': 'Return Successd!', 'result': 'NAME ERROR!', 'resultcode': '201'}}\n\n    @pytest.mark.process\n    @pytest.mark.parametrize('data',testdata)\n    def test_constellation_info(self, data):\n        self.log.info('%s{%s}' % ((sys._getframe().f_code.co_name, '------开始星座运势查询接口测试-----')))\n        with allure.step('星座运势查询接口'):\n            msg = Api_News_Service().get_constellation(data['data']['consName'])\n        self.log.info('%s:%s' % ((sys._getframe().f_code.co_name, '获取请求结果：%s' % msg.json())))\n        # # 断言\n        assert data['resp']['result'] in (str)(msg.json()), '关键字\"{}\"不存在源码中!'.format(msg.json())\n        assert msg.json()['resultcode'] == data['resp']['resultcode']\n>       assert msg.json()['error_code'] == data['resp']['error_code']\nE       assert 205801 == 0\n\nTestCase\\test_constellation_info.py:29: AssertionError",
    "steps" : [ {
      "name" : "星座运势查询接口",
      "time" : {
        "start" : 1630848282695,
        "stop" : 1630848282726,
        "duration" : 31
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "shouldDisplayMessage" : false,
      "attachmentsCount" : 0,
      "stepsCount" : 0,
      "hasContent" : false
    } ],
    "attachments" : [ {
      "uid" : "9bd1178ed8bccd99",
      "name" : "log",
      "source" : "9bd1178ed8bccd99.txt",
      "type" : "text/plain",
      "size" : 434
    } ],
    "parameters" : [ ],
    "shouldDisplayMessage" : true,
    "attachmentsCount" : 1,
    "stepsCount" : 1,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "feature",
    "value" : "聚合接口"
  }, {
    "name" : "tag",
    "value" : "process"
  }, {
    "name" : "parentSuite",
    "value" : "TestCase"
  }, {
    "name" : "suite",
    "value" : "test_constellation_info"
  }, {
    "name" : "subSuite",
    "value" : "Test_Service_Info"
  }, {
    "name" : "host",
    "value" : "DESKTOP-9PO3806"
  }, {
    "name" : "thread",
    "value" : "11072-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "TestCase.test_constellation_info"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "data",
    "value" : "{'detail': '异常测试：星座字段空', 'data': {'consName': '', 'type': 'today'}, 'resp': {'resultcode': '201', 'reason': 'Return Successd!', 'result': 'NAME ERROR!', 'error_code': 0}}"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "538ee578f24bdcd9",
      "status" : "failed",
      "statusDetails" : "AssertionError: assert 205801 == 0",
      "time" : {
        "start" : 1630848217027,
        "stop" : 1630848217058,
        "duration" : 31
      }
    }, {
      "uid" : "4a6f8270456e01f1",
      "status" : "failed",
      "statusDetails" : "AssertionError: assert 205801 == 0",
      "time" : {
        "start" : 1630848150603,
        "stop" : 1630848150635,
        "duration" : 32
      }
    }, {
      "uid" : "d8f83864a8289a3c",
      "status" : "failed",
      "statusDetails" : "AssertionError: assert 205801 == 0",
      "time" : {
        "start" : 1630848003191,
        "stop" : 1630848003222,
        "duration" : 31
      }
    }, {
      "uid" : "27b1842ae4c4e39d",
      "status" : "failed",
      "statusDetails" : "AssertionError: assert 205801 == 0",
      "time" : {
        "start" : 1630847811847,
        "stop" : 1630847811878,
        "duration" : 31
      }
    }, {
      "uid" : "6ff557e116523975",
      "status" : "failed",
      "statusDetails" : "AssertionError: assert 205801 == 0",
      "time" : {
        "start" : 1630847759185,
        "stop" : 1630847759216,
        "duration" : 31
      }
    } ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "process" ]
  },
  "source" : "8038a4fd5d6af28f.json",
  "parameterValues" : [ "{'detail': '异常测试：星座字段空', 'data': {'consName': '', 'type': 'today'}, 'resp': {'resultcode': '201', 'reason': 'Return Successd!', 'result': 'NAME ERROR!', 'error_code': 0}}" ]
}